gcamwrapper/query_library.py

```diff
@@ -0,0 +1,251 @@
+import yaml
+import warnings
+import re
+import os.path as path
```

Comment:
Due to third party libs that have similar functions, it is better to just import `os` and call `os.path` in your code.

Reply:
Thanks, I think this isn't even used anymore since switching to using `pkg_resources` but I'll keep this in mind in the future.